File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>
int arr[1000000];
int main(void) {
	int N;
	int max = -1000000;
	int min = 1000000;
	scanf("%d", &N);
	if (1 <= N && N <= 1000000) {
		for (int i = 0; i < N; i++) {
			scanf("%d", &arr[i]);
			if (arr[i] < min) {
				min = arr[i];
			}
			if (arr[i] > max) {
				max = arr[i];
			}
		}
	}
	printf("%d %d", min, max);
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
N = int(input())
L = list(map(int, input().split()))
max = L[0]
min = L[0]
if(1 <= N) and (N <= 1000000):
    for i in L:
        if i > max:
            max = i
        if i < min:
            min = i
print(min, max)